aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -116,6 +116,26 @@ If opting to remove a module with a setting in *web.config*, unlock the module a
     </system.webServer>
    </configuration>
    ```
+   
+In order to manipulate modules for IIS Express using *web.config*, you must modify *applicationHost.config* to unlock the `<modules>` section:
```

Comment:
In order to manipulate modules for IIS Express using *web.config*, *applicationHost.config* must be modified to unlock the `<modules>` section: